### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by JALsnipe**
- [X] Add .gitignore
- [X] Remove pycache directory
- [X] Add requirements.txt
- [X] Changes chromedriver path

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-CHROMEDRIVER_PATH = "D:\Desktop\ppdg_egc_extractor\chromedriver.exe"
```
Comment (JALsnipe) [inline config.py]: Remove this change

**Comment by JALsnipe** [comment]
I will fix all merge conflicts and push a new commit.

**Comment by JALsnipe** [comment]
@Churn2Burn please review and merge.

File: requirements.txt [hunk 2]
```diff
@@ -1,4 +1,5 @@
-requests
-bs4
-selenium
-image
\ No newline at end of file
+# TODO: lock these into specific versions
```
Comment (Churn2Burn) [inline requirements.txt]: Please remove.

**Review by Churn2Burn — CHANGES_REQUESTED** [review]

**Review by Churn2Burn — APPROVED** [review]

**Pull request merged**